.github/workflows/cf-deploy-react.yml

```diff
@@ -21,20 +22,24 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          cache: 'yarn'
           node-version: 18
       - run: corepack enable
-      - run: yarn set version stable
+      - uses: actions/setup-node@v4
+        with:
+          cache: yarn
+          node-version: 18
       - run: yarn install --immutable
+      - run: yarn build:widget
       - run: yarn run lint:react-widget
       - run: yarn run build:react-widget
```

Comment:
if you build all packages, you can remove this. You should also remove lint as it's being run on the ci workflow